src/types/commitData.ts

```diff
@@ -6,3 +6,16 @@ export type GitHubCommitData = {
   }
   message: string
 }
+
+export type GitLocalDiskCommitData = {
+  sha: string
+} & GitHubCommitData
```

Comment:
Hmm I think it would be better if GitHubCommitData has the `sha` attribute added (plus a simple modification to GitHubService, GitHub already provides this field) since we should make GitHubService and GitFileSystemService return the same types as much as possible.

Reply:
Sure!

Reply:
but I think sha can be an optional attribute within GitHubCommitData

Reply:
done!